packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/RelationInputDataManager.js

```diff
@@ -141,13 +140,10 @@ export const RelationInputDataManger = ({
       description={description}
       disabled={isDisabled}
       id={name}
-      label={formatMessage(
-        {
-          id: intlLabel.id,
-          defaultMessage: `${intlLabel.defaultMessage} ({numberOfEntries})`,
-        },
-        { numberOfEntries: relationsCount }
-      )}
+      label={`${formatMessage({
+        id: intlLabel.id,
+        defaultMessage: intlLabel.defaultMessage,
+      })} ${initialData[name]?.count ? `(${initialData[name].count})` : ''}`}
```

Comment:
If `count` equals 0 it won't display it no?

What about 
```js
})} ${initialData[name]?.count !== undefined ? `(${initialData[name].count})` : ''}`}
```
or
```js
const { has } = require('lodash/fp');

// ...

})} ${has('count', initialData[name]) ? `(${initialData[name].count})` : ''}`}
```

Reply:
indeed good catch, thanks 🙂